static/js/components/CourseCarousel.js

```diff
@@ -16,50 +17,66 @@ type Props = {|
   setShowResourceDrawer: Function
 |}
 
-const prevButton = ({ previousSlide, currentSlide }) =>
-  currentSlide === 0 ? null : (
-    <div onClick={previousSlide} className="carousel-control prev">
-      <i className="material-icons">chevron_left</i>
-    </div>
-  )
+export default function CourseCarousel(props: Props) {
+  const { title, courses, setShowResourceDrawer } = props
 
-const nextButton = props => {
-  const { slideCount, nextSlide, currentSlide, slidesToShow } = props
-  const lastSlideIndex = slideCount - 1
+  const [index, setIndex] = useState(0)
+  const canPageUp = index + CAROUSEL_PAGE_SIZE < courses.length
+  const canPageDown = index !== 0
 
-  const disabled = currentSlide + slidesToShow > lastSlideIndex
-  return disabled ? null : (
-    <div onClick={nextSlide} className="carousel-control next">
-      <i className="material-icons">chevron_right</i>
+  return (
+    <div className="course-carousel">
+      <div className="title-and-controls">
+        <div className="title">{title}</div>
+        <div className="controls">
+          <button
+            className={`dark-outlined compact extra-compact prev ${
```

Comment:
What do you think of moving these buttons a bit closer to the carousel?  It can be a bit hard to tell which buttons belong to which carousel if you don't already know they should be on top:

![Screen Shot 2019-09-11 at 11 31 09 AM](https://user-images.githubusercontent.com/187676/64711491-b1319180-d487-11e9-84ef-7f61062f76d8.png)

Reply:
are you sure you pulled down the most recent version? the padding below the buttons should look like it does in the screenshots above

Reply:
Just pulled again, looks good now 👍